engine-tests/src/test/java/org/terasology/documentation/BindingScraper.java

```diff
@@ -19,6 +21,7 @@
  * Enumerates all default key bindings and writes them sorted by ID to the console
  */
 public final class BindingScraper {
+    private static final Logger LOGGER = LoggerFactory.getLogger(BindingScraper.class);
```

Comment:
See my comment above. This also looks like a utility tool and the class-level comment strongly suggests that the output format (without a logger) is intentional.

Reply:
hmm. you are right @BenjaminAmos . we try to get docmentation via these, i found : https://github.com/MovingBlocks/Terasology/issues/1975 ? @Cervator ?

Reply:
reverted.